qwikidata/sparql.py

```diff
@@ -19,9 +23,17 @@ def return_sparql_query_results(
     wikidata_sparql_url: str, optional
       wikidata SPARQL endpoint to use
     """
-    return requests.get(
+    resp = requests.get(
         wikidata_sparql_url, params={"query": query_string, "format": "json"}
-    ).json()
+    )
+    if resp.status_code != 200:
+        raise SPARQLResponseNotOk(
```

Comment:
I think I'd probably prefer the caller to use a try/except or maybe something like this here (https://docs.python-requests.org/en/latest/user/quickstart/#response-status-codes) ... what do you thing?